pyro/infer/autoguide/guides.py

```diff
@@ -44,16 +44,14 @@ class AutoGuide(ConstrainedModule):
     :class:`AutoGuideList` object.
 
     :param callable model: a pyro model
-    :param str prefix: a prefix that will be prefixed to all param internal sites
-        as part of the module name.
     """
 
-    def __init__(self, model, prefix="auto"):
+    def __init__(self, model):
         super().__init__()
         self.master = None
         # Do not register model as submodule
         self._model = (model,)
-        self.prefix = prefix
+        self._prefix = "auto"
```

Comment:
This is to keep things backwards compatible, we can also expose this attribute but it will only be used for the master guide.